### PR TITLE
docs: Updated README.md file for Devtron chart

### PR DESCRIPTION
**Pull request by Shubham9t9**

### CODE REVIEW

File: charts/devtron/README.md [hunk 1]
```diff
@@ -24,32 +24,23 @@ It packages third party components like
 
 ### Install with Helm
 
-This chart is currently not available on the official helm repository therefore you need to download it to install it.
+Run the following command to install the latest version of Devtron along with the CI/CD module:
```
Comment (SNe789) [inline charts/devtron/README.md:27]: Run the following command to install the latest version of Devtron along with CI/CD integration:

File: charts/devtron/README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ It packages third party components like
  - [Argocd](https://github.com/argoproj/argo-cd/) for gitops 
  - [Argo workflows](https://github.com/argoproj/argo) for CI
  - [Clair](https://github.com/quay/clair) & [Guard](https://github.com/guard/guard) for image scanning
- - [Kubernetes External Secrets](https://github.com/godaddy/kubernetes-external-secrets) for ingegrating with external secret management stores like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [HashiCorp Vault](https://www.vaultproject.io/)
+ - [External Secret Operator](https://github.com/external-secrets/external-secrets) and [Kubernetes External Secrets (Depricated) ](https://github.com/godaddy/kubernetes-external-secrets) for ingegrating with external secret management stores like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [HashiCorp Vault](https://www.vaultproject.io/)
```
Comment (SNe789) [inline charts/devtron/README.md:18]: ....for integrating with external secret management systems like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [HashiCorp Vault](https://www.vaultproject.io/)

File: charts/devtron/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 # Devtron Installation
 
-Devtron is an open source software delivery workflow for kubernetes written in go. It is designed as a self-serve platform for operationalizing and maintaining applications (AppOps) on kubernetes in a developer friendly way.
+Devtron is an open source software delivery workflow for Kubernetes written in Go. It is designed as a self-serve platform for operationalizing and maintaining applications (AppOps) on Kubernetes in a developer friendly way.
```
Comment (SNe789) [inline charts/devtron/README.md:6]: ....written in Go programming language.

File: charts/devtron/README.md [hunk 4]
```diff
@@ -24,32 +24,23 @@ It packages third party components like
...
+--create-namespace --namespace devtroncd \
+--set installer.modules={cicd}
 ```
-For more details about configuration see the [helm chart configuration](#configuration)
 
-### Install with kubectl
+[Link to](https://docs.devtron.ai/install) more installation options for Devtron.
 
-If you don't want to install helm on your cluster and just want to use `kubectl` to install `devtron platform`, then please follow following steps:
+For more details about configuration see the [Helm chart configuration](#configuration)
```
Comment (SNe789) [inline charts/devtron/README.md:39]: For more details on configuration, see the [Helm chart configuration](#configuration).

**Review by SNe789 — COMMENTED** [review]
Added comments.

File: charts/devtron/README.md (hunk 1 above)
Comment (Shubham9t9) [reply charts/devtron/README.md:27]: done

File: charts/devtron/README.md (hunk 2 above)
Comment (Shubham9t9) [reply charts/devtron/README.md:18]: done

File: charts/devtron/README.md (hunk 3 above)
Comment (Shubham9t9) [reply charts/devtron/README.md:6]: done

File: charts/devtron/README.md (hunk 4 above)
Comment (Shubham9t9) [reply charts/devtron/README.md]: done

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**